client/src/test/java/com/alibaba/nacos/client/logging/log4j2/Log4J2NacosLoggingTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.alibaba.nacos.client.logging.log4j2;
+
+import org.junit.Test;
+
+public class Log4J2NacosLoggingTest {
+    
+    @Test
+    public void testLoadConfiguration() {
+        Log4J2NacosLogging log4J2NacosLogging = new Log4J2NacosLogging();
+        log4J2NacosLogging.loadConfiguration();
```

Comment:
No assert or verify?

Reply:
currently I do not have a good way to verify  nacos-logback.xml have been loaded. So I verify exception do not throw out.
JUnit4 do not have assert for notexception